ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/authoring/css/authoring.css

```diff
@@ -13,7 +13,3 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  ******************************************************************************/
-
-[data-cmp-visible="false"] {
```

Comment:
This modification is backward in-comatible. Ideally, we should address the root issue in the original code instead of fixing the CSS. Removing this data attribute could impact custom core components that rely on it, and not doing so may result in their hidden components remaining hidden rather than being visible during authoring.